UNRELEASED.md

```diff
@@ -12,6 +12,8 @@ Use [the changelog guidelines](https://git.io/polaris-changelog-guidelines) to f
 
 ### Bug fixes
 
+- Fix `Banner`’s `id` server/client mismatch ([#3199](https://github.com/Shopify/polaris-react/pull/3199)).
```

Comment:
Moving from a class to a functional component is probs not worth an entry in `code quality`?